synapse/storage/__init__.py

```diff
@@ -128,16 +128,21 @@ def persist_event(self, event, context, backfilled=False,
             pass
 
     @defer.inlineCallbacks
-    def get_event(self, event_id, allow_none=False):
-        events = yield self._get_events([event_id])
+    def get_event(self, event_id, check_redacted=True,
+                  get_prev_content=False, allow_rejected=False,
+                  allow_none=False):
```

Comment:
Might want to add a bit of __doc__ string for the get_event now that it has a few parameters.